Design Hashmap.java

```diff
@@ -0,0 +1,27 @@
+// Time Complexity : O(1)
+// Space Complexity :O(n)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : Yes
+
+
+// Your code here along with comments explaining your approach
```

Comment:
Implement your hash map by doing hashing on key, you don't have to create array of integers of size 1000001

Reply:
Use hashing on given key for get(), put() and remove() methods.